### DISCUSSION TITLE
[feature request] Add header to block Floc

### DISCUSSION THREAD

**Discussion by awerner25** [nginx-proxy/nginx-proxy#1622] [Ideas]
Hello,

It's possible to add an option to add a certain header in each server{} section.
`add_header Permissions-Policy interest-cohort=();`

[to block google floc](https://paramdeo.com/blog/opting-your-website-out-of-googles-floc-network)

**Comment by tkw1536** [comment]
This is already possible by [Replacing default proxy settings](https://github.com/nginx-proxy/nginx-proxy#replacing-default-proxy-settings).

Concretely you could extend the default settings by adding the following content inside `/etc/nginx/proxy.conf` inside the nginx-proxy container:

```
# default settings

# HTTP 1.1 support
proxy_http_version 1.1;
proxy_buffering off;
proxy_set_header Host $http_host;
proxy_set_header Upgrade $http_upgrade;
proxy_set_header Connection $proxy_connection;
proxy_set_header X-Real-IP $remote_addr;
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
proxy_set_header X-Forwarded-Proto $proxy_x_forwarded_proto;
proxy_set_header X-Forwarded-Ssl $proxy_x_forwarded_ssl;
proxy_set_header X-Forwarded-Port $proxy_x_forwarded_port;

# Mitigate httpoxy attack (see README for details)
proxy_set_header Proxy "";

# /default settings

# any custom headers here
add_header Permissions-Policy interest-cohort=();
````
